dags/copy_deduplicate.py

```diff
@@ -105,6 +105,21 @@
         ],
     )
 
+    with TaskGroup(group_id='main_ping_downstream_external') as main_ping_downstream_external:
+        ExternalTaskMarker(
+            task_id="graphics_telemetry_graphics_trends",
+            external_dag_id="graphics_telemetry",
+            external_task_id="graphics_trends",
+        )
+
+        ExternalTaskMarker(
+            task_id="graphics_telemetry_graphics_dashboard",
+            external_dag_id="graphics_telemetry",
+            external_task_id="graphics_dashboard",
+        )
+
+    copy_deduplicate_main_ping >> main_ping_downstream_external
+
```

Comment:
I'm curious if/how this will show up in the default tree view, as the [docs](https://airflow.apache.org/docs/apache-airflow/2.1.4/concepts/dags.html#taskgroups) only mention it affecting the _graph_ view.

TaskGroups also automatically prefix the task IDs with `{group_id}.` which is handy in this case I think.

Reply:
Yeah, I'm also curious. We use `SubDag` in some places which only shows one box in the tree view and lets you expand into the sub tasks. However the docs say `SubDag` is deprecated and to use task groups instead.